Dockerfile

```diff
@@ -6,4 +6,4 @@ COPY contrib/gitlab.tpl contrib/gitlab.tpl
 COPY contrib/junit.tpl contrib/junit.tpl
 COPY contrib/sarif.tpl contrib/sarif.tpl
 USER appuser
-ENTRYPOINT ["trivy"]
+ENTRYPOINT ["trivy"]
```

Comment:
Could you revert that if there is no reason you want to remove the newline at end of file?